### ISSUE TITLE
How do you use /cfunction

### ISSUE THREAD

**Issue by VFox32** [Earthcomputer/clientcommands#554]
I was making some functions and came across the /cfunction command. I assumed it worked like regular functions, however, i cant find any documentation of the command and how to use it anywhere.

**Comment by Coderx-Gamer** [comment]
I'm pretty sure it does just work the same as normal server-side functions. I checked the code for it, looks like the functions are stored in .minecraft/config/clientcommands/functions .

**Comment by VFox32** [comment]
@Coderx-Gamer Ok, i found the folder and it recognizes the files in-game, but when i try to run one it says "Unknown function <file path>"

**Comment by Earthcomputer** [comment]
Does the function come up in auto-complete?

**Comment by VFox32** [comment]
yes it shows up in auto complete and when i try to execute it, it gives the error "Unknown Function (file path)". 
Also, sorry for taking so long to respond xD gotto get used to checking my e-mails every day

**Comment by m-sarabi** [comment]
I added the wiki page for the [/cfunction](../wiki/cfunction) command, including examples. I hope this is helpful. Note that you can only run client-side commands with it.

**Comment by VFox32** [comment]
Hello there. its been a while, but i once again wanted to use the cfunction command.
i made a file in the folder clientcommands/functions called test.mcfunction...] 
when i try to run it, it does show up in autocomplete within minecraft, but says "unknown or incomplete command [...] cfunction test<--[HERE]". so basically exactly the same as i said before in this thread.
safe to say i have no clue why its not working for me...

**Comment by xpple** [comment]
Could you send your `test.mcfunction` file?

**Comment by VFox32** [comment]
well, its called `test.mcfunction` and in it i put `ctellraw "test"`
i would upload the file but github says no to .mcfunction xD

edit: omfg i just "fixed" it. turns out it can only run clientcommands commands. in the wiki it did say only clientside commands, but i figured it would work in singleplayer world with everything, since that is technically also client, though i guess its run on a local/virtual server (you know what i mean) but of well, at least i figured it out...

edit2: its a bit sad though. no cfunction for quickly spawning bots using carpet :(
